app/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
```

Comment:
This is result of Visual Code formatting tool. We decided to use it and result looks ok to me.